packages/app/public/learning-paths/data.json

```diff
@@ -272,5 +272,77 @@
     "description": "For the best experience in this learning path, we suggest that you complete the following learning resources in the order shown. When you click on a resource, it will open in a new tab. Keep this page open so you can easily move on to the next resource!",
     "label": "Using projects in Red Hat OpenShift Data Science",
     "url": "https://developers.redhat.com/learn/openshift-data-science/using-projects-red-hat-openshift-data-science"
+  },
+  {
+    "minutes": 30,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Learn more about the fundamental concepts of Ansible, an open source IT automation solution that automates provisioning, configuration management, application deployment, orchestration, and many other IT processes.",
+    "label": "Introduction to Ansible",
+    "url": "https://red.ht/aap-lp-ansible-basics"
+  },
+  {
+    "hours": 1,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Do a deep-dive into the features of Ansible VS Code extension, the Red Hat recommended way of creating your Ansible projects, which brings syntax-highlighting, auto-completion, linting, and generative AI to your automation creation workflows.",
+    "label": "Getting started with the Ansible VS Code extension",
+    "url": "https://red.ht/aap-lp-vscode-essentials"
+  },
+  {
+    "minutes": 30,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Learn more about YAML and how Ansible Automation Platform uses it for content. YAML offers a straightforward, human-readable way to write Ansible configurations, tasks, and playbooks.",
+    "label": "YAML Essentials for Ansible",
+    "url": "https://red.ht/aap-lp-yaml-essentials"
+  },
+  {
+    "hours": 1,
+    "level": "Beginner",
+    "type": "Ansible Learning path",
+    "description": "Learn how to create Ansible playbooks, the blueprints for automation tasks executed across an inventory of nodes. Playbooks tell Ansible which tasks to perform on which devices.",
+    "label": "Getting started with Ansible playbooks",
+    "url": "https://red.ht/aap-lp-getting-started-playbooks"
```

Comment:
This link also redirects to https://developers.redhat.com/products/ansible/overview